src/Sulu/Bundle/LocationBundle/Resources/config/geolocator.xml

```diff
@@ -2,26 +2,22 @@
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
-        <parameter key="sulu_location.guzzle.client.class">GuzzleHttp\Client</parameter>
         <parameter key="sulu_location.geolocator.nominatim.class">Sulu\Bundle\LocationBundle\Geolocator\Service\NominatimGeolocator</parameter>
         <parameter key="sulu_location.geolocator.google.class">Sulu\Bundle\LocationBundle\Geolocator\Service\GoogleGeolocator</parameter>
     </parameters>
 
     <services>
-        <!-- Guzzle client -->
-        <service id="sulu_location.geolocator.guzzle.client" class="%sulu_location.guzzle.client.class%"/>
-
         <!-- Geolocator services -->
         <!-- * Nominatim * -->
         <service id="sulu_location.geolocator.service.nominatim" class="%sulu_location.geolocator.nominatim.class%">
-            <argument type="service" id="sulu_location.geolocator.guzzle.client" />
+            <argument type="service" id="Symfony\Contracts\HttpClient\HttpClientInterface" />
             <argument>%sulu_location.geolocator.service.nominatim.endpoint%</argument>
             <argument>%sulu_location.geolocator.service.nominatim.api_key%</argument>
         </service>
 
         <!-- * Google * -->
         <service id="sulu_location.geolocator.service.google" class="%sulu_location.geolocator.google.class%">
-            <argument type="service" id="sulu_location.geolocator.guzzle.client" />
+            <argument type="service" id="Symfony\Contracts\HttpClient\HttpClientInterface" />
```

Comment:
Bundles should not use aliases, and so in his case use `http_client` here as service definition.